### PR TITLE
WIP: Improve CoreData Documentation

### PR DESCRIPTION
**Pull request by mkistler**
WIP: improved documentation as per issue #40.

### CODE REVIEW

File: doc/coredata.md [hunk 1]
```diff
@@ -16,37 +18,21 @@ this.
...
-```objc
-- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
-
-    [CDTIncrementalStore initialize];
-	...
-
-```
-
-> ***Note***: it is safe to call `+initialize` multiple times.
+If you have not already done so, add the CDTDatastore CocoaPod to your project.
```
Comment (jimix) [inline doc/coredata.md:23]: Mention that the instructions are in the README, perhaps with a link?

File: doc/coredata.md [hunk 2]
```diff
@@ -148,7 +132,30 @@ while (!done) {
...
+The Cloudant persistent store for CoreData supports the following features:
+
+- Save, fetch, update, and delete of Managed Objects to the on-device CDTDatastore.
+- Batch update requests
+- Asynchronous fetch operations
+- Schema migration
+
+## Unsupported Features
+
+The following features of CoreData are currently not supported by the CDTIncrementalStore:
```
Comment (jimix) [inline doc/coredata.md:146]: link?

**Comment by jimix** [comment]
Added `WIP:` prefix to the title so I can see it from the PR summary page

**Comment by mkistler** [comment]
Ancient history

**Pull request closed without merging**